src/img_web/app/views.py

```diff
@@ -52,10 +51,10 @@ def submit(request):
         reposformset = extraReposFormset()
         tokensformset = extraTokensFormset()
         ppformset = postProcessFormset()
-        return render_to_response('app/upload.html',
+        return render (request, 'app/upload.html',
 
                                   {'jobform' : jobform, 'reposformset' : reposformset, 'tokensformset' : tokensformset,
-                                   'ppformset' : ppformset}, context_instance=RequestContext(request)
+                                   'ppformset' : ppformset}, context=RequestContext(request).flatten()
```

Comment:
This is not right, the dictionary defined here should go to the context parameter, and you don't need to have RequestContext here, the render function does that automatically now from the given request object.

This applies to all render_to_response -> render changes.